projects/plugins/backup/src/php/class-jetpack-backup.php

```diff
@@ -158,8 +158,17 @@ public function register_rest_routes() {
 			)
 		);
 
+		// Get information on site products.
+		register_rest_route(
+			'jetpack/v4',
+			'/site/purchases',
```

Comment:
We could use a unique route here, with the intention to revert to this, in a later version, once the functionality is extracted into a common package.  Currently we're using the same route as the jetpack plugin, and depending on which route gets called while both plugins are active a change to either could break the other.